app/controllers/locations_controller.rb

```diff
@@ -0,0 +1,5 @@
+class LocationsController < ApplicationController
+  def index
+    @locations = Locations.near_postcode(params['postcode'])
```

Comment:
We're being very trusting of the `Locations` class, either here or app/views/locations/index.html.erb#2 needs an `Array` coercion
```ruby
@locations = Array(Locations.near_postcode(params['postcode']))
```